client/ayon_core/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -71,11 +71,12 @@ class HoudiniSubmitDeadline(
     order = pyblish.api.IntegratorOrder
     hosts = ["houdini"]
     families = ["usdrender",
```

Comment:
Guess we need to pack the rop(s) into one published family before the farm submission? It seems to me that it is still submitting to the render farm even if the user chooses the local render.

Reply:
If that's the case - that's just because `instance.data["farm"]` apparently does not get set? Since this plug-in should get skipped by the `instance.data["farm"]` check in `process()`

Reply:
> If that's the case - that's just because `instance.data["farm"]` apparently does not get set? Since this plug-in should get skipped by the `instance.data["farm"]` check in `process()`

This can be the way to solve this. I believe it is added into the submit_publish_job if I remember correctly.

Reply:
> Guess we need to pack the rop(s)-related families into one published family before the farm submission?

This was done in https://github.com/ynput/ayon-core/pull/328/commits/941e80dd952864adcd4ba4386d5883434f5cd338 but I found out that we lost the attribute_defs in the publisher as BigRoy mentioned here https://github.com/ynput/ayon-core/pull/328#discussion_r1544893280 
So, I had to remove it in order to get the attribute_defs back. 

Also, the `instance.data["farm"]` check was added to `process()`. 
Hopefully, This skips the job submission.